pkgs/build-support/fetchurl/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, curl }: # Note that `curl' may be `null', in case of the native stdenv.
+{ stdenv, curl, ca_cert_file ? null, insecure ? false }: # Note that `curl' may be `null', in case of the native stdenv.
```

Comment:
Was there a discussion about (dis)advantages of using insecure connection by default for fixed-output derivations? For most use cases I don't think there's any added security, and I suppose there's some (small) performance loss.